linux_sql/scripts/host_info.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+#assigning CLI arguments to variables
+
+psql_host=$1
+psql_port=$2
+db_name=$3
+psql_user=$4
+psql_password=$5
+
+#getting hardware information of the host
+
+id=1
+hostname=$(hostname -f)
+cpu_number=$(lscpu | grep -E "^CPU\(s\):" | awk '{print $2}')
+cpu_architecture=$(lscpu | grep "Architecture" | awk '{print $2}')
+cpu_model=$(lscpu | grep "Model:" | awk '{print $2}')
+cpu_mhz=$(lscpu | grep "CPU MHz:" | awk '{print $3}')
+l2_cache=$(lscpu | grep "L2 cache:" | awk '{print $3}' | sed 's/K//')
+total_mem=$(cat /proc/meminfo | grep "MemAvailable:" | awk '{print $2}')
+timestamp=$(vmstat -t | awk 'NR==3{print $(NF-1), $NF}')
+
+#installing psql CLI client
+
+sudo yum install -y postgresql
```

Comment:
remove yum install as we can assume psql cli is already installed

BTW, remove extra empty lines